test/selenium/README.md

```diff
@@ -22,18 +22,106 @@ Run the [run-tests script](https://github.com/cockpit-project/cockpit/blob/maste
+    ```
+  - Run hub and attach browsers here:
+    ```
+    java -jar selenium.jar -role hub
+    ```
+    - Then attach your local browsers there. You have browsers eg. firefox or chrome installed, and also ensure you have proper [selenium drivers for selected the browser](https://github.com/cockpit-project/cockpit/tree/master/test/selenium#which-browser-to-use)
+      ```
+      java -jar selenium.jar -role node -hub https://localhost:4444/grid/register
+      ```
+  - Run the test. ``HUB_IP`` is ip address of your selenium grid
+    ```
+    test/selenium/run-tests --hub $HUB_IP -v -b chrome selenium-base.py --sit
+    ```
+#### Reschedule the the test/s again
+It is possible to reschedule tests if you used ``--sit`` option for ``run-tests``, in case of failure, machines won't be destroyed.
+Command-line options depend on which selenium grid you used.
+It is described on the output of the ``run-tests`` script. Eg.:
+
+```
+$test/selenium/run-tests --hub $HUB_IP -v -b chrome selenium-navigate.py --sit
+
+...
+To rerun tests connect to VM machine with avocado installed via
+    ssh -p 2202 root@127.0.0.2
+
+and run the command
+    PYTHONPATH=/tmp/avocado_library HUB=$HUB_IP_ADDR GUEST=127.0.0.2
+PORT=9091 SSH_PORT=22 SSH_GUEST=10.111.113.1 BROWSER=chrome timeout 297
+python3 -m avocado run --show-job-log /tmp/avocado_tests/selenium-navigate.py:NavigateTestSuite.testNavigateNoReload 2>&1
+```
+
+It is possible also to run the tests on your machine, without test VM (bots/images) what contains installed avocado testing framework.
+  - You have to install python3 ``avocado`` via rpm or pip
+  - Then a little bit modify command what you see on the output of the ``run-tests`` command and it depends how you run the selenium grid.
+    ```
+    HUB=$HUB_IP GUEST=$GUEST_IP PORT=9991 SSH_PORT=2021 SSH_GUEST=127.0.0.2 BROWSER=chrome python3 -m avocado --show=test run  test/selenium/selenium-navigate.py
+    ```
+   - **Remember**
+     - ``GUEST`` has to be accessible from ``HUB`` via ``PORT`` number
+     - Your avocado runner has to be able to connect ``SSH_GUEST`` via ``SSH_PORT`` (what is same machine as ``GUEST`` but may have another address, because of port forwarding and ``HUB`` location)
+
+##### Description of example output
+
+  - ``PYTHONPATH=/tmp/avocado_library`` - path to libraries
+  - ``HUB=$HUB_IP_ADDR`` - ip address of your selenium grid
+  - ``GUEST=127.0.0.2`` - ip adress of cockpit machine (``HUB`` has to see ``GUEST`` cockpit port)
+  - ``PORT=9091`` - port of cockpit machine on ``GUEST`` ip adress
+  - ``SSH_PORT=22`` - ssh port number on ``SSH_GUEST``
+  - ``SSH_GUEST=10.111.113.1`` - guest ip how to connect to ssh (``GUEST`` has to be able to connect ``SSH_GUEST:SSH_PORT``)
+  - ``BROWSER=chrome`` - name of browser to use
+  - ``timeout 297`` - timeout to kill the ``avocado`` process (could be remove in case of manual rescheduling)
```

Comment:
Missing equal after the timeout word.
And s/remove/removed/

Reply:
@KKoukiou there is not missing equal, it is bash ``command`` not env ``variable``

I've introduced it a long time ago, to avoid killing subprocess by python,  but kill it explicitly, to show output of process instead of traceback. without information